src/components/CreateItem.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import firebase from 'firebase/app';
+
+function CreateItem() {
+  const handleClick = async () => {
+    await firebase.firestore().collection('items').add({
+      name: 'A test product',
+      date: Date.now(),
```

Comment:
Grabbing `Date.now()` is, from what I can tell, a good way to grab the current timestamp in some cases.

> The static Date.now() method returns the number of milliseconds elapsed since January 1, 1970 00:00:00 UTC. - [Date.now()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/now)

However, if we're storing that timestamp as a number, there's a potential that it may be a little more difficult to work with than you'd expect, with respect to timezones and daylight savings and so on. Basically, working with time is stupidly hard and wherever possible, lean on someone else to handle the time. In this case, it looks like firebase has a `Timestamp` type, and we could use it to get handling time off of our plate. It also comes with many other benefits like easily making time calculations and comparisons. They've already done all of the hard work here, so let's let them do it.

I'd encourage you to check out [firebase.firestore.Timestamp](https://firebase.google.com/docs/reference/js/v9/firestore_.timestamp).

Reply:
That’s a very good point about handling timezones. And thank you for telling us about `Timestamp` from firebase, we’ll try that.

Reply:
This is a great suggestion for `future user` stories as well 👏